Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiVer3.cs

```diff
@@ -40,8 +42,8 @@ public ObservableList<ApplicationAPIModel> OpenApiThree(SwaggerDocument Swaggerd
         {
             
             opendoc = Swaggerdoc;
-            
-            
+
+            var listExampleValues = GetExamplesFromOpenApiComponents(opendoc.Components);
```

Comment:
Please move this code inside the SetOptionalValue function to make it more generic

Reply:
Moving inside the SetOptionalValue func was not feasible, implemented this GetExamplesFromOpenApiComponents func call whenever SetOptionalValue func will be called as one of its parameter.